lib/liftoff/project_configuration.rb

```diff
@@ -20,7 +20,8 @@ class ProjectConfiguration
       :xcode_command,
       :extra_config,
       :extra_test_config,
-      :deployment_target
+      :deployment_target,
+      :schemes
```

Comment:
Yeah, what you linked to is a method definition. It's actually a syntax error there so we can't do it. The example that shows this case is [here](https://github.com/thoughtbot/guides/blob/master/style/ruby/sample.rb#L65)